### DISCUSSION TITLE
FLYPIE BUG on GNOME

### DISCUSSION THREAD

**Discussion by ghost** [Schneegans/Fly-Pie#311] [Fly-Pie Q&A]
Hi all, every time I open the flypie menu, it ALWAYS terminates after a few seconds. Please review issue #310 

This is extremely irritating and renders the entire extension almost useless. Any help would be greatly appreciated!

**Comment by Schneegans** [comment]
Hi there! There is no need to ask the same question multiple times. Once I find the time, I'll answer. You know - I am doing this in my _spare time_! I have a full-time job, kids, and it is holiday season.

**Reply by ghost** [reply]
I'm very sorry, Simon. I've been trying to find the root of the issue and it has been devastatingly difficult to figure out what is causing the bug. I do not believe that it is something in the core functionality of FlyPie, because I have never experienced this before. Maybe the GSlice error is something that is related to my GNOME libraries (maybe glibc was changed...)

At any rate, thank you for responding. Please take all the time you need, and enjoy the next few weeks. If there is anything that you would like for me to look into, or data that you would like for me to collect, please do not hesitate to ask. Thanks for all of your contributions.
